spec/system/home_page_spec.rb

```diff
@@ -37,8 +37,7 @@
     
     expect(page).to have_css('.radio-group.download-radio-buttons__radio-group > li.radio-group__item', count: 5)
 
-    # Can be flaky because of the dependence on the very specific `for` attribute
-    page.find('label[for="download-radio-buttons-56-option-saltmarshes"] > .radio-button__radio').click
+    page.find('label[for*="saltmarshes"] > .radio-button__radio').click
```

Comment:
Just a teeny one: Would `downloads_dialog.find('label[for*=...` work here (can you do that?)? I guess it's feasible that in the future other inputs for the habitats could be added to the page, which could mess with this test.